airflow/www/views.py

```diff
@@ -108,8 +108,13 @@ def get_safe_url(url):
     valid_schemes = ['http', 'https', '']
     valid_netlocs = [request.host, '']
 
+    # Remove single quotes
+    url = url.replace("'", "")
```

Comment:
What's this?

Reply:
fixed in https://github.com/apache/airflow/pull/12459/commits/7486153f451e4d2bb1c6fd9cbb5a63430157c99c